### PR TITLE
Move pagination page size to config object

### PR DESCRIPTION
**Pull request by robyoung**
I'm not sure if this is the right thing to do. I'm not completely against it being set in the module but it had a TODO so I took a stab at addressing it. Thoughts?

### CODE REVIEW

File: app/main/views/services.py [hunk 1]
```diff
@@ -55,8 +51,11 @@ def list_services():
 
         services = services.filter(Service.supplier_id == supplier_id)
 
-    services = services.paginate(page=page, per_page=API_FETCH_PAGE_SIZE,
-                                 error_out=False)
+    services = services.paginate(
+        page=page,
+        per_page=current_app.config['DM_SUPPLIERS_PAGE_SIZE'],
```
Comment (allait) [inline app/main/views/services.py]: Should be `DM_API_SUPPLIERS_PAGE_SIZE`

File: app/main/views/suppliers.py [hunk 2]
```diff
@@ -27,7 +24,7 @@ def list_suppliers():
 
     suppliers = suppliers.paginate(
         page=page,
-        per_page=API_FETCH_PAGE_SIZE,
+        per_page=current_app.config['DM_SUPPLIERS_PAGE_SIZE'],
```
Comment (allait) [inline app/main/views/suppliers.py]: Should be `DM_API_SUPPLIERS_PAGE_SIZE`

**Comment by TheDoubleK** [comment]
Yep, we definitely want separate pagination sizes for services and suppliers I reckon.

**Comment by TheDoubleK** [comment]
I'm not against having the values set as code - I don't think it's something that needs to be particularly configurable.  Maybe include default values in the code that can be overridden by config if necessary?

**Pull request merged**